Added hasParents WHSSD PEV

MaaikevS opened the pull request:
All the PEV now have a parent.

For some reason it also updated one of the JBA files (I guess I was behind of a commit from someone else). I checked and the 'changes' are identical to what was already done, so this should not cause a problem.

MaaikevS committed:
Added hasParent

MaaikevS committed:
Added hasParent

MaaikevS commented:
@UlrikeS91 Please have look at this! Thank you!

UlrikeS91 commented:
> All the PEV now have a parent.
> 
> For some reason it also updated one of the JBA files (I guess I was behind of a commit from someone else). I checked and the 'changes' are identical to what was already done, so this should not cause a problem.

@lzehl, could you also check that this one JBA file would be ok to merge in the end? I'm not sure why this ended up in her PR. If it was already merged in to the main branch it shouldn't show up, right? 

I will check the WHS files and content. But already now, thanks for the work @MaaikevS!

lzehl commented on instances/atlas/parcellationEntityVersion/WHSSD_v1-01/WHSSD_v1-01_cingulateCortexArea2.jsonld:
```diff
@@ -22,7 +22,11 @@
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_cerebralCortexIncludingTheNeocortexAndTheHippocampus"
```
is there another cerebralCortex ? if not please shorten this id to the necessary part

lzehl commented on instances/atlas/parcellationEntityVersion/WHSSD_v1-01/WHSSD_v1-01_dorsalintermediateEntorhinalArea.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
```
same here. if possible shorten the id

lzehl commented on instances/atlas/parcellationEntityVersion/WHSSD_v1-01/WHSSD_v1-01_dorsallateralEntorhinalArea.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
```
see other comments on id

lzehl commented on instances/atlas/parcellationEntityVersion/WHSSD_v1-01/WHSSD_v1-01_frontalAssociationCortex.jsonld:
```diff
@@ -22,7 +22,11 @@
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_cerebralCortexIncludingTheNeocortexAndTheHippocampus"
```
same here as other comments for id

lzehl commented on instances/atlas/parcellationEntityVersion/WHSSD_v1-01/WHSSD_v1-01_hippocampalFormation.jsonld:
```diff
@@ -22,7 +22,11 @@
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_cerebralCortexIncludingTheNeocortexAndTheHippocampus"
```
same as other comments for id

lzehl commented on instances/atlas/parcellationEntityVersion/WHSSD_v1-01/WHSSD_v1-01_medialEntorhinalField.jsonld:
```diff
@@ -22,7 +22,11 @@
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_cerebralCortexIncludingTheNeocortexAndTheHippocampus"
```
same as other comments on id

lzehl commented on instances/atlas/parcellationEntityVersion/WHSSD_v1-01/WHSSD_v1-01_neocortex.jsonld:
```diff
@@ -22,7 +22,11 @@
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_cerebralCortexIncludingTheNeocortexAndTheHippocampus"
```
same comment on id

lzehl commented on instances/atlas/parcellationEntityVersion/WHSSD_v1-01/WHSSD_v1-01_perirhinalCortex.jsonld:
```diff
@@ -22,7 +22,11 @@
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_cerebralCortexIncludingTheNeocortexAndTheHippocampus"
```
shorten if possible

lzehl commented on instances/atlas/parcellationEntityVersion/WHSSD_v1-01/WHSSD_v1-01_postrhinalCortex.jsonld:
```diff
@@ -22,7 +22,11 @@
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_cerebralCortexIncludingTheNeocortexAndTheHippocampus"
```
shorten if possible

lzehl commented on instances/atlas/parcellationEntityVersion/WHSSD_v1-01/WHSSD_v1-01_subiculum.jsonld:
```diff
@@ -22,7 +22,11 @@
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_cerebralCortexIncludingTheNeocortexAndTheHippocampus"
```
shorten if possible

lzehl commented on instances/atlas/parcellationEntityVersion/WHSSD_v1-01/WHSSD_v1-01_ventralintermediateEntorhinalArea.jsonld:
```diff
@@ -22,7 +22,11 @@
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_cerebralCortexIncludingTheNeocortexAndTheHippocampus"
```
shorten if possible

lzehl commented on instances/atlas/parcellationEntityVersion/WHSSD_v2/WHSSD_v2_cingulateCortexArea2.jsonld:
```diff
@@ -22,7 +22,11 @@
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_cerebralCortexIncludingTheNeocortexAndTheHippocampus"
```
shorten if possible

lzehl commented on instances/atlas/parcellationEntityVersion/WHSSD_v2/WHSSD_v2_cornuAmmonis1.jsonld:
```diff
@@ -22,7 +22,11 @@
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_cerebralCortexIncludingTheNeocortexAndTheHippocampus"
```
shorten if possible

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v1-01/WHSSD_v1-01_facialNerve.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_facialNerve"
+        }
+    ],
     "lookupLabel": "WHSSD_v1.01_facialNerve",
     "name": "facial nerve",
```
"facial nerve" with parent "facial nerve" is a bit annoying. Nothing wrong with your script, @MaaikevS. It is exactly what the atlas files say. 
Maybe we should take the liberty and rename the PEV to "facial nerve, unspecified"?

lzehl reviewed (commented):
@MaaikevS thanks a lot for this PR :)

I checked the JBA file (it was only one if I saw correct) and it took over the changes that were made to it already. I also assume that you missed a commit in your branch compared to v3. 

There is one (repeating comment) for a parent structure with a super long `@id`. This is a comment not on your work but just occurred to me now. @UlrikeS91 if you want to keep the `@id` as it is for maintainability reasons and not shorten it please say so (it's fine with me; @MaaikevS in this case please ignore these comments).

Everything else looks good. I went through 90 and they look formally correct and also in the content logic (I did not went to double check the parent structures; @UlrikeS91 I assume you discussed with Maaike and they are correct?).

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v1-01/WHSSD_v1-01_glomerularLayerOfTheAccessoryOlfactoryBulb.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_olfactoryBulb"
+        }
+    ],
```
mbat file says "olfactory system" as parent. I remember we talked about the olfactory bulb, just need to check again if this makes sense or if I told you something wrong.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v1-01/WHSSD_v1-01_glomerularLayerOfTheOlfactoryBulb.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_olfactoryBulb"
+        }
+    ],
```
see comment for "glomerular layer of the accessory olfactory bulb"

MaaikevS commented:
@lzehl I merged the commits before making my changes, but I think indeed that this one slipped through the cracks..

I am happy to share or attached the overview file that the parents are based on if you would like to have this as a reference.

Regarding the other comments, let me know if you decide to change anything and I will change accordingly!

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v1-01/WHSSD_v1-01_innerEar.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_grayMatter"
+        }
+    ],
```
This is wrong. It on the same hierarchical level as gray matter. The v1.01 mbat file does not have a higher level. So, either we add "brain" as parent (if this has been handled like this in the following versions) or it does not get a parent.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v1-01/WHSSD_v1-01_medialLemniscus.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_medialLemniscus"
+        }
+    ],
     "lookupLabel": "WHSSD_v1.01_medialLemniscus",
     "name": "medial lemniscus",
```
"medial lemniscus" with parent "medial lemniscus" is a bit annoying. Nothing wrong with your script, @MaaikevS. It is exactly what the atlas files say.
Maybe we should take the liberty and rename the PEV to "medial lemniscus, unspecified"?

lzehl commented:
@MaaikevS no need for the overview file. You discussed this with Ulrike, correct (and she said she is double checking)? If @UlrikeS91 gives her "ignore" for my comments I will merge the PR

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v1-01/WHSSD_v1-01_olfactoryBulb.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_olfactoryBulb"
+        }
+    ],
     "lookupLabel": "WHSSD_v1.01_olfactoryBulb",
     "name": "olfactory bulb",
```
"olfactory bulb" with parent "olfactory bulb" is a bit annoying. Also see comment for "glomerular layer of the accessory olfactory bulb". mbat file states "olfactory system" as parent. Need to check this again.

MaaikevS commented:
Of course! Just making sure :)

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v1-01/WHSSD_v1-01_ventricularSystem.jsonld:
```diff
@@ -22,7 +22,11 @@
+        }
+    ],
     "lookupLabel": "WHSSD_v1.01_ventricularSystem",
     "name": "ventricular system",
```
This is wrong. Similar comment as for "inner ear". Same hierarchical level as "gray matter". mbat for v1.01 has no higher level. So, either we add "brain" as parent (if this has been handled like this in the following versions) or it does not get a parent.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v2/WHSSD_v2_anteriorCommissure.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_anteriorCommisure"
+        }
+    ],
     "lookupLabel": "WHSSD_v2_anteriorCommissure",
     "name": "anterior commissure",
```
"anterior commissure" with parent "anterior commissure" is a bit annoying. Nothing wrong with your script, @MaaikevS. It is exactly what the atlas files say even though they corrected themselves in the mbat to say "anterior commissure, intrabulbar part".
Maybe we should take the liberty and rename the PEV to either "anterior commissure, intrabulbar part" (and have what they have in the mbat) or "anterior commissure, unspecified"?

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v1-01/WHSSD_v1-01_centralCanal.jsonld:
```diff
@@ -22,7 +22,11 @@
+        }
+    ],
     "lookupLabel": "WHSSD_v1.01_centralCanal",
     "name": "central canal",
```
This is wrong. Should be "spinal cord". This is some sort of error in your script, @MaaikevS.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v1-01/WHSSD_v1-01_spinalCord.jsonld:
```diff
@@ -22,7 +22,11 @@
+        }
+    ],
     "lookupLabel": "WHSSD_v1.01_spinalCord",
     "name": "spinal cord",
```
This is wrong. "spinal cord" has "spinal cord" as parent according to mbat file - bit annoying. 
Maybe we should take the liberty and rename the PEV to "spinal cord, unspecified"?

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v2/WHSSD_v2_centralCanal.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_ventricularSystem"
+        }
+    ],
```
This is wrong. Should be "spinal cord". This is some sort of error in your script, @MaaikevS.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v2/WHSSD_v2_deeperLayersOfTheSuperiorColliculus.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_corticofugalPathways"
+        }
+    ],
```
This is wrong. Should be "tectum". This is some sort of error in your script, @MaaikevS.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v2/WHSSD_v2_dentateGyrus.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_tectum"
+        }
+    ],
```
This is wrong. Should be "Cerebral cortex including the neocortex and the hippocampus". This is some sort of error in your script, @MaaikevS.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v2/WHSSD_v2_descendingCorticofugalPathways.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_cerebralCortexIncludingTheNeocortexAndTheHippocampus"
+        }
+    ],
```
This is wrong. Should be "Corticofugal pathways". This is some sort of error in your script, @MaaikevS.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v2/WHSSD_v2_entorhinalCortex.jsonld:
```diff
@@ -22,7 +22,11 @@
+        }
+    ],
     "lookupLabel": "WHSSD_v2_entorhinalCortex",
     "name": "entorhinal cortex",
```
v2 has "lateral entorhinal cortex " & "entorhinal cortex" but on same hierarchical level, which means that "entorhinal cortex" should most likely be "entorhinal cortex, unspecified" or "medial entorhinal cortex". Should check if we should rename it.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v2/WHSSD_v2_facialNerve.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_facialNerve"
+        }
+    ],
     "lookupLabel": "WHSSD_v2_facialNerve",
     "name": "facial nerve",
```
Same comment as for v1.01. "facial nerve" with parent "facial nerve" is a bit annoying. Nothing wrong with your script, @MaaikevS. It is exactly what the atlas files say.
Maybe we should take the liberty and rename the PEV to "facial nerve, unspecified"?

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v2/WHSSD_v2_glomerularLayerOfTheAccessoryOlfactoryBulb.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_olfactoryBulb"
+        }
+    ],
     "lookupLabel": "WHSSD_v2_glomerularLayerOfTheAccessoryOlfactoryBulb",
     "name": "glomerular layer of the accessory olfactory bulb",
```
Same as v1.01 comment. mbat says "olfactory system". need to check.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v2/WHSSD_v2_glomerularLayerOfTheOlfactoryBulb.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_olfactoryBulb"
+        }
+    ],
     "lookupLabel": "WHSSD_v2_glomerularLayerOfTheOlfactoryBulb",
     "name": "glomerular layer of the olfactory bulb",
```
Same as v1.01 comment. mbat says "olfactory system". need to check.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v2/WHSSD_v2_innerEar.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_grayMatter"
+        }
+    ],
```
Same comment as v1.01. Wrong. Either "brain" or nothing. Need to check.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v2/WHSSD_v2_medialLemniscus.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_medialLemniscus"
+        }
+    ],
     "lookupLabel": "WHSSD_v2_medialLemniscus",
     "name": "medial lemniscus",
```
Same comment as for v1.01. "medial lemniscus" with parent "medial lemniscus" is a bit annoying. Nothing wrong with your script, @MaaikevS. It is exactly what the atlas files say.
Maybe we should take the liberty and rename the PEV to "medial lemniscus, unspecified"?

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v2/WHSSD_v2_olfactoryBulb.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_olfactoryBulb"
+        }
+    ],
     "lookupLabel": "WHSSD_v2_olfactoryBulb",
     "name": "olfactory bulb",
```
Same comment as v1.01. "olfactory bulb" with parent "olfactory bulb" is a bit annoying. Also see comment for "glomerular layer of the accessory olfactory bulb". mbat file states "olfactory system" as parent. Need to check this again.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v2/WHSSD_v2_spinalCord.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_ventricularSystem"
+        }
+    ],
```
Same comment as for v1.01. Wrong. parent is "spinal cord", should consider renaming PEV to "spinal cord, unspecified"

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v2/WHSSD_v2_ventricularSystem.jsonld:
```diff
@@ -22,7 +22,11 @@
+        }
+    ],
     "lookupLabel": "WHSSD_v2_ventricularSystem",
     "name": "ventricular system",
```
Same comment as for v1.01. Wrong. Either "brain" or nothing. need to check.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3-01/WHSSD_v3-01_acousticStriae.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_cerebellarAndPrecerebellarWhiteMatter"
+        }
+    ],
```
Wrong. Should be "White matter of the brainstem".

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3-01/WHSSD_v3-01_facialNerve.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_facialNerve"
+        }
+    ],
     "lookupLabel": "WHSSD_v3.01_facialNerve",
     "name": "facial nerve",
```
Same comment as for v1.01 and v2. Maybe rename PEV to "facial nerve, unspecified"?

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3-01/WHSSD_v3-01_glomerularLayerOfTheAccessoryOlfactoryBulb.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_olfactoryBulb"
+        }
+    ],
     "lookupLabel": "WHSSD_v3.01_glomerularLayerOfTheAccessoryOlfactoryBulb",
     "name": "glomerular layer of the accessory olfactory bulb",
```
Similar as for v1.01 and v2. mbat states "olfactory areas" as parent. Need to check.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3-01/WHSSD_v3-01_glomerularLayerOfTheOlfactoryBulb.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_olfactoryBulb"
+        }
+    ],
     "lookupLabel": "WHSSD_v3.01_glomerularLayerOfTheOlfactoryBulb",
     "name": "glomerular layer of the olfactory bulb",
```
Similar as for v1.01 and v2. mbat states "olfactory areas" as parent. Need to check.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3-01/WHSSD_v3-01_lateralLemniscus.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_lateralLemniscus"
+        }
+    ],
     "lookupLabel": "WHSSD_v3.01_lateralLemniscus",
     "name": "lateral lemniscus",
```
"lateral lemniscus" with parent ""lateral lemniscus". Should consider renaming PEV to "lateral lemniscus, unspecified"

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3-01/WHSSD_v3-01_medialGeniculateBodyDorsalDivision.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_medialGeniculateComplexOfTheDorsalThalamus"
+        }
+    ],
     "lookupLabel": "WHSSD_v3.01_medialGeniculateBodyDorsalDivision",
     "name": "medial geniculate body, dorsal division",
```
Similar comment as for facial nerve. I think we talked about it, but now I'm unsure. Will check again if it needs to be changed to "medial geniculate body" ad mbat for v3.01 says.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3-01/WHSSD_v3-01_medialGeniculateBodyMarginalZone.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_medialGeniculateComplexOfTheDorsalThalamus"
+        }
+    ],
     "lookupLabel": "WHSSD_v3.01_medialGeniculateBodyMarginalZone",
     "name": "medial geniculate body, marginal zone",
```
Similar comment as for facial nerve. I think we talked about it, but now I'm unsure. Will check again if it needs to be changed to "medial geniculate body" ad mbat for v3.01 says.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3-01/WHSSD_v3-01_medialGeniculateBodyMedialDivision.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_medialGeniculateComplexOfTheDorsalThalamus"
+        }
+    ],
     "lookupLabel": "WHSSD_v3.01_medialGeniculateBodyMedialDivision",
     "name": "medial geniculate body, medial division",
```
Similar comment as for facial nerve. I think we talked about it, but now I'm unsure. Will check again if it needs to be changed to "medial geniculate body" ad mbat for v3.01 says.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3-01/WHSSD_v3-01_medialGeniculateBodyVentralDivision.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_medialGeniculateComplexOfTheDorsalThalamus"
+        }
+    ],
     "lookupLabel": "WHSSD_v3.01_medialGeniculateBodyVentralDivision",
     "name": "medial geniculate body, ventral division",
```
Similar comment as for facial nerve. I think we talked about it, but now I'm unsure. Will check again if it needs to be changed to "medial geniculate body" ad mbat for v3.01 says.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3-01/WHSSD_v3-01_medialLemniscus.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_medialLemniscus"
+        }
+    ],
     "lookupLabel": "WHSSD_v3.01_medialLemniscus",
     "name": "medial lemniscus",
```
"medial lemniscus" with parent "medial lemniscus". Maybe change PEV to "medial lemniscus, unspecified"

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3-01/WHSSD_v3-01_olfactoryBulb.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_olfactoryBulb"
+        }
+    ],
     "lookupLabel": "WHSSD_v3.01_olfactoryBulb",
     "name": "olfactory bulb",
```
same as for other olfactory areas - need to check if it should be "olfactory areas"

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3-01/WHSSD_v3-01_reticularThalamicNucleusAuditorySegment.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_reticularPrethalamicNucleus"
+        }
+    ],
     "lookupLabel": "WHSSD_v3.01_reticularThalamicNucleusAuditorySegment",
     "name": "reticular thalamic nucleus, auditory segment",
```
Same as olfactory areas. I think I said to do this. Unsure now...Will check again.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3-01/WHSSD_v3-01_spinalCord.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_brain"
+        }
+    ],
```
Wrong. On same hierarchical level as "brain". Remove parent - has none.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3-01/WHSSD_v3-01_spinalTrigeminalTract.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_cerebellarAndPrecerebellarWhiteMatter"
+        }
+    ],
```
Wrong. Should be "White matter of the brainstem".

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3-01/WHSSD_v3-01_trapezoidBody.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_cerebellarAndPrecerebellarWhiteMatter"
+        }
+    ],
```
Wrong. Should be "White matter of the brainstem".

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3/WHSSD_v3_anteriorCommissure.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_anteriorCommisure"
+        }
+    ],
     "lookupLabel": "WHSSD_v3_anteriorCommissure",
     "name": "anterior commissure",
```
"anterior commissure" with parent "anterior commissure". Was fixed in 3.01, not here. Should consider rename to "anterior commissure, intrabulbar part" (as in mbat file) or "anterior commissure, unspecified".

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3/WHSSD_v3_basalForebrainRegion.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_cerebrum"
+        }
+    ],
     "lookupLabel": "WHSSD_v3_basalForebrainRegion",
     "name": "basal forebrain region",
```
Is "basal forebrain region, unspecified". Fixed in v3.01, not here.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3/WHSSD_v3_brainstem.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_brainstem"
+        }
+    ],
     "lookupLabel": "WHSSD_v3_brainstem",
     "name": "brainstem",
```
"brainstem" with parent "brainstem". Should consider renaming PEV to "brainstem, unspecified" (also what they wrote in mbat file). Fixed in v3.01, not here.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3/WHSSD_v3_entorhinalCortex.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_entorhinalCortex"
+        }
+    ],
     "lookupLabel": "WHSSD_v3_entorhinalCortex",
     "name": "entorhinal cortex",
```
"entorhinal cortex" with parent "entorhinal cortex". Should consider renaming PEV to "medial entorhinal cortex" (as they also state in mbat file). Fixed in v3.01, not here.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3/WHSSD_v3_facialNerve.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_facialNerve"
+        }
+    ],
     "lookupLabel": "WHSSD_v3_facialNerve",
     "name": "facial nerve",
```
"facial nerve" with parent "facial nerve". Should consider renaming PEV to "facial nerve, unspecified"?

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3/WHSSD_v3_glomerularLayerOfTheAccessoryOlfactoryBulb.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_olfactoryBulb"
+        }
+    ],
     "lookupLabel": "WHSSD_v3_glomerularLayerOfTheAccessoryOlfactoryBulb",
     "name": "glomerular layer of the accessory olfactory bulb",
```
same as other versions. need to check if "olfactory areas" should be used

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3/WHSSD_v3_glomerularLayerOfTheOlfactoryBulb.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_olfactoryBulb"
+        }
+    ],
     "lookupLabel": "WHSSD_v3_glomerularLayerOfTheOlfactoryBulb",
     "name": "glomerular layer of the olfactory bulb",
```
same as other versions. need to check if "olfactory areas" should be used

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3/WHSSD_v3_lateralLemniscus.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_lateralLemniscus"
+        }
+    ],
     "lookupLabel": "WHSSD_v3_lateralLemniscus",
     "name": "lateral lemniscus",
```
"lateral lemniscus" with parent "lateral lemniscus". Consider renaming PEV to "lateral lemniscus, unspecified". Same issue in v3.01.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3/WHSSD_v3_mammillothalamicTract.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_thalamicTracts"
+        }
+    ],
     "lookupLabel": "WHSSD_v3_mammillothalamicTract",
     "name": "mammillothalamic tract",
```
Is "mammillotegmental tract". Fixed in v3.01, not here.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3/WHSSD_v3_medialGeniculateBodyDorsalDivision.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_medialGeniculateComplexOfTheDorsalThalamus"
+        }
+    ],
     "lookupLabel": "WHSSD_v3_medialGeniculateBodyDorsalDivision",
     "name": "medial geniculate body, dorsal division",
```
same as v3.01, need to check, maybe "medial geniculate body" instead

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3/WHSSD_v3_medialGeniculateBodyMarginalZone.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_medialGeniculateComplexOfTheDorsalThalamus"
+        }
+    ],
     "lookupLabel": "WHSSD_v3_medialGeniculateBodyMarginalZone",
     "name": "medial geniculate body, marginal zone",
```
same as v3.01, need to check, maybe "medial geniculate body" instead

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3/WHSSD_v3_medialGeniculateBodyMedialDivision.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_medialGeniculateComplexOfTheDorsalThalamus"
+        }
+    ],
     "lookupLabel": "WHSSD_v3_medialGeniculateBodyMedialDivision",
     "name": "medial geniculate body, medial division",
```
same as v3.01, need to check, maybe "medial geniculate body" instead

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3/WHSSD_v3_medialGeniculateBodyVentralDivision.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_medialGeniculateComplexOfTheDorsalThalamus"
+        }
+    ],
     "lookupLabel": "WHSSD_v3_medialGeniculateBodyVentralDivision",
     "name": "medial geniculate body, ventral division",
```
same as v3.01, need to check, maybe "medial geniculate body" instead

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3/WHSSD_v3_medialLemniscus.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_medialLemniscus"
+        }
+    ],
     "lookupLabel": "WHSSD_v3_medialLemniscus",
     "name": "medial lemniscus",
```
"medial lemniscus" with parent "medial lemniscus". Consider renaming PEV to "medial lemniscus, unspecified". Same issue in v3.01.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3/WHSSD_v3_neocortex.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_isocortex"
+        }
+    ],
     "lookupLabel": "WHSSD_v3_neocortex",
     "name": "neocortex",
```
"neocortex, unspecified" in v3.01, not here.  Maybe change, since mbat also says it.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3/WHSSD_v3_olfactoryBulb.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_olfactoryBulb"
+        }
+    ],
     "lookupLabel": "WHSSD_v3_olfactoryBulb",
     "name": "olfactory bulb",
```
"olfactory bulb" with parent "olfactory bulb", need to check and maybe change to "olfactory areas"

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3/WHSSD_v3_reticularThalamicNucleusAuditorySegment.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_reticularPrethalamicNucleus"
+        }
+    ],
     "lookupLabel": "WHSSD_v3_reticularThalamicNucleusAuditorySegment",
     "name": "reticular thalamic nucleus, auditory segment",
```
same comment as v3.01, need to check and maybe change to "reticular nucleus of the thalamus"

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3/WHSSD_v3_spinalCord.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_brain"
+        }
+    ],
```
Wrong. Same hierarchical level. Remove parent - has none.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3/WHSSD_v3_spinalTrigeminalTract.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_cerebellarAndPrecerebellarWhiteMatter"
+        }
+    ],
```
Wrong. Should be "White matter of the brainstem".

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3/WHSSD_v3_thalamus.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_thalamus"
+        }
+    ],
     "lookupLabel": "WHSSD_v3_thalamus",
     "name": "thalamus",
```
"thalamus" with parent "thalamus". Consider renaming PEV to "thalamus, unspecified" (as mbat says). Fixed in v3.01, not here.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3/WHSSD_v3_trapezoidBody.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_cerebellarAndPrecerebellarWhiteMatter"
+        }
+    ],
```
Wrong. Should be "White matter of the brainstem".

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3/WHSSD_v3_ventralPeriolivaryNuclei.jsonld:
```diff
@@ -22,7 +22,11 @@
+        }
+    ],
     "lookupLabel": "WHSSD_v3_ventralPeriolivaryNuclei",
     "name": "ventral periolivary nuclei",
```
"ventral periolivary nucleus" in v3.01 label and mbat; here onlt in mbat called that. label has "ventral periolivary nuclei"

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v3/WHSSD_v3_ventricularSystem.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_ventricularSystem"
+        }
+    ],
     "lookupLabel": "WHSSD_v3_ventricularSystem",
     "name": "ventricular system",
```
"ventricular system" with parent "ventricular system". Consider renaming PEV to "ventricular system, unspecified" (as the mbat already says). Fixed in v3.01, not here.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v4/WHSSD_v4_angularThalamicNucleus.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_ventralMidlineGroupOfTheDorsalThalamus"
+        }
+    ],
```
Wrong. Should be "ventral nuclei of the dorsal thalamus".

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v4/WHSSD_v4_commissuralStriaTerminalis.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_mammillotegmentalTract"
+        }
+    ],
```
Wrong. Should be "thalamic tracts".

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v4/WHSSD_v4_ethmoidLimitansNucleus.jsonld:
```diff
@@ -22,7 +22,11 @@
+        }
+    ],
     "lookupLabel": "WHSSD_v4_ethmoidLimitansNucleus",
     "name": "ethmoid-Limitans nucleus",
```
Should this be "Ethmoid-Limitans nucleus" or "ethmoid-limitans nucleus" or is the captialisation as it is correct?

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v4/WHSSD_v4_fasciculusRetroflexus.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_mammillotegmentalTract"
+        }
+    ],
```
Wrong. Should be "thalamic tracts".

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v4/WHSSD_v4_frontalAssociationArea3.jsonld:
```diff
@@ -22,7 +22,11 @@
         "visualizedIn": null
     },
-    "hasParent": null,
+    "hasParent": [
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_motorcortex"
+        }
+    ],
     "lookupLabel": "WHSSD_v4_frontalAssociationArea3",
     "name": "frontal association area 3",
```
It's what it says in the files, but is this actually correct?

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v4/WHSSD_v4_habenularCommissure.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_mammillotegmentalTract"
+        }
+    ],
```
Wrong. Should be "thalamic tracts".

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v4/WHSSD_v4_intermediodorsalThalamicNucleus.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_dorsallateralEntorhinalArea"
+        }
+    ],
```
Wrong. Should be "Dorsal-caudal midline group of the dorsal thalamus".

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v4/WHSSD_v4_internalMedullaryLamina.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_thalamicTracts"
+        }
+    ],
```
Wrong. Should be "white matter".

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v4/WHSSD_v4_intramedullaryThalamicArea.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_thalamicTracts"
+        }
+    ],
```
Wrong. Should be "white matter".

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v4/WHSSD_v4_mammillotegmentalTract.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_whiteMatter"
+        }
+    ],
```
Wrong. Should be "thalamic tracts".

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v4/WHSSD_v4_parataenialThalamicNucleus.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_dorsallateralEntorhinalArea"
+        }
+    ],
```
Wrong. Should be "dorsal-caudal midline group of the dorsal thalamus"

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v4/WHSSD_v4_paraventricularThalamicNucleiAnteriorAndPosterior.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_dorsallateralEntorhinalArea"
+        }
+    ],
```
Wrong. Should be "dorsal-caudal midline group of the dorsal thalamus"

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v4/WHSSD_v4_posteriorIntralaminarNucleus.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_dorsallateralEntorhinalArea"
+        }
+    ],
```
Wrong. Should be "dorsal-caudal midline group of the dorsal thalamus"

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v4/WHSSD_v4_retroreuniensThalamicNucleus.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_ventralintermediateEntorhinalArea"
+        }
+    ],
```
Wrong. Should be "ventral midline group of the dorsal thalamus".

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v4/WHSSD_v4_reuniensThalamicNucleus.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_ventralintermediateEntorhinalArea"
+        }
+    ],
```
Wrong. Should be "ventral midline group of the dorsal thalamus".

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v4/WHSSD_v4_rhomboidThalamicNucleus.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_ventralintermediateEntorhinalArea"
+        }
+    ],
```
Wrong. Should be "ventral midline group of the dorsal thalamus".

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v4/WHSSD_v4_spinalCord.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_brain"
+        }
+    ],
```
Wrong. Same hierarchical level. Remove parent - has none.

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v4/WHSSD_v4_striaMedullarisThalami.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_mammillotegmentalTract"
+        }
+    ],
```
Wrong. Should be "thalamic tracts".

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v4/WHSSD_v4_striaTerminalis.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_mammillotegmentalTract"
+        }
+    ],
```
Wrong. Should be "thalamic tracts".

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v4/WHSSD_v4_submediusThalamicNucleus.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_ventralMidlineGroupOfTheDorsalThalamus"
+        }
+    ],
```
Wrong. Should be "ventral nuclei of the dorsal thalamus".

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v4/WHSSD_v4_subparafascicularNucleus.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_dorsallateralEntorhinalArea"
+        }
+    ],
```
Wrong. Should be "dorsal-caudal midline group of the dorsal thalamus"

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v4/WHSSD_v4_ventralAnteriorThalamicNucleus.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_ventralMidlineGroupOfTheDorsalThalamus"
+        }
+    ],
```
Wrong. Should be "ventral nuclei of the dorsal thalamus".

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v4/WHSSD_v4_ventralPosteriorNucleusOfTheThalamusParvicellularPart.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_ventralPosteriorNucleusOfTheThalamusParvicellularPart"
+        }
+    ],
```
Wrong. Should be "ventral posterior thalamic nucleus"

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v4/WHSSD_v4_ventralPosterolateralThalamicNucleus.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_ventralPosteriorNucleusOfTheThalamusParvicellularPart"
+        }
+    ],
```
Wrong. Should be "ventral posterior thalamic nucleus"

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v4/WHSSD_v4_ventralPosteromedialThalamicNucleus.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_ventralPosteriorNucleusOfTheThalamusParvicellularPart"
+        }
+    ],
```
Wrong. Should be "ventral posterior thalamic nucleus"

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v4/WHSSD_v4_ventrolateralThalamicNucleus.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_ventralMidlineGroupOfTheDorsalThalamus"
+        }
+    ],
```
Wrong. Should be "ventral nuclei of the dorsal thalamus"

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v4/WHSSD_v4_ventromedialThalamicNucleus.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_ventralMidlineGroupOfTheDorsalThalamus"
+        }
+    ],
```
Wrong. Should be "ventral nuclei of the dorsal thalamus"

UlrikeS91 commented on instances/atlas/parcellationEntityVersion/WHSSD_v4/WHSSD_v4_xiphoidThalamicNucleus.jsonld:
```diff
@@ -22,7 +22,11 @@
+        {
+            "@id": "https://openminds.ebrains.eu/instances/parcellationEntity/WHSSD_ventralintermediateEntorhinalArea"
+        }
+    ],
```
Wrong. Should be "ventral midline group of the dorsal thalamus"

UlrikeS91 commented:
> @MaaikevS thanks a lot for this PR :)

Yes, thanks again! I went through everything now and commented on everything that I need to check or that is definitely wrong. 

> I checked the JBA file (it was only one if I saw correct) and it took over the changes that were made to it already. I also assume that you missed a commit in your branch compared to v3.

Okay, good. Better save than sorry. So, this can be ignored. 

> There is one (repeating comment) for a parent structure with a super long `@id`. This is a comment not on your work but just occurred to me now. @UlrikeS91 if you want to keep the `@id` as it is for maintainability reasons and not shorten it please say so (it's fine with me; @MaaikevS in this case please ignore these comments).

I'd like to keep it as is for maintainability reasons. This is what the PE was called in the first versions of the atlas. Later on they are split up into smaller areas but it will get very complicated if we removed this long one...
 
> Everything else looks good. I went through 90 and they look formally correct and also in the content logic (I did not went to double check the parent structures; @UlrikeS91 I assume you discussed with Maaike and they are correct?).

Formally everything is correct, yes. Content logic is also mostly correct, but there are some errors here and there. Additionally, I think that I caused some things myself where I told @MaaikevS to do it that way but now that I see it here and could think about it, I get unsure again 🙈 Sorry about that...

UlrikeS91 commented:
@MaaikevS, as you can see, I had comments about 91 files (or so). How would you like to proceed with this? There are several ones that are wrong and need to be changed (and I think I also always comments "wrong"). But the rest is not as straight forward...Some comments would require us to deviate away from the "ground-truth" of the atlas versions and I need to check some others again because I'm not sure anymore if we should solve it that way. 

What do you think? Do you have any preferences?

MaaikevS commented:
@UlrikeS91 Thanks for commenting.
I noticed that the ones that are linked to the wrong parents originate from a stupid filter in the excel file. Apologies for creating unnecessary work! I will commit a new version, which should solve this issue.

Regarding the olfactary bulb vs olfactory system/olfactory areas, this is indeed what we discussed, but if you would like to change it, let me know!

The comment about renaming instances because the parent has the same name or there is another reason to consider this, I will let you and others decide on this. This is also not related to the link to a parent parcellation entity, so I think we should move that to a different discussion.

MaaikevS committed:
Fixed hasParent

MaaikevS commented:
Fixed the parents for the following files according to your comments, @UlrikeS91. I removed the parent for inner ear in v1 and v2, and for spinal cord in v3, v3-01 and v4.
![image](https://user-images.githubusercontent.com/31585951/153385465-7e1d6ac1-e496-48f0-b98f-07b7939cf908.png)

Please get back to me regarding the olfactory system and what you would like to do with it. I will start a new issue for the re-naming of instances

lzehl commented:
@MaaikevS is this ready to be reviewed again by @UlrikeS91 ?

MaaikevS commented:
@lzehl Yes, I changed what was requested. @UlrikeS91 can review this again and will also need to get back to me in case the olfactory bulb parent should be changed. I have done everything on my part.

UlrikeS91 commented:
Yes, I'm sorry. With the schema changes, I first wanted to check what we may need to change in addition to the changes here or if something needs to be done differently now. @MaaikevS, I will get back to you about it. I have not forgotten about it, but I need a bit more time and we may have to have a meeting offline to discuss.

MaaikevS committed:
Fixed hasParent

MaaikevS committed:
Update WHSSD_v4_spinalCord.jsonld

MaaikevS commented:
@UlrikeS91 This should now be fixed.
